initial_pose_grid_x: 0
initial_pose_grid_y: 0
initial_pose_cart_x: 0.25
initial_pose_cart_y: 0.25